cmd/eksctl-anywhere/cmd/installpackagecontroller.go

```diff
@@ -7,21 +7,24 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
 	"github.com/aws/eks-anywhere/pkg/curatedpackages"
 	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/utils/urls"
+	"github.com/aws/eks-anywhere/pkg/validations"
 	"github.com/aws/eks-anywhere/pkg/version"
 )
 
 type installControllerOptions struct {
-	kubeVersion string
+	fileName string
 }
 
 var ico = &installControllerOptions{}
 
 func init() {
 	installCmd.AddCommand(installPackageControllerCommand)
-	installPackageControllerCommand.Flags().StringVar(&ico.kubeVersion, "kube-version", "", "Bundle version to use")
-	if err := installPackageControllerCommand.MarkFlagRequired("kube-version"); err != nil {
+	installPackageControllerCommand.Flags().StringVarP(&ico.fileName, "filename", "f", "", "Filename that contains EKS-A cluster configuration")
```

Comment:
This is a breaking change, so do we want to go through a deprecation path where we give a warning message and then slowly remove it after 2 releases? Or are we okay with the consequences?

Reply:
We still haven't gone GA, I think this should be fine but I'll check with the team.

Reply:
Checked with the team and the deprecation process is not necessary. I'll update the docs to reflect this change though.